antora-ui-camel/src/css/header.css

```diff
@@ -394,6 +399,18 @@ body {
   text-align: right;
 }
 
+.search-bar {
+  top: 4.1rem;
+  position: sticky;
+  position: -webkit-sticky;
```

Comment:
Yes, we need this 'cause chrome tends to behave in a weird manner at times compared to the other browsers such as firefox and safari. Hence, both the position is required.

Reply:
Should be the same as `sticky`, it can be used unprefixed according to [caniuse.com](https://caniuse.com/#feat=css-sticky) -- 91.35% browsers don't require the prefix.